Formula/b2sum.rb

```diff
@@ -0,0 +1,20 @@
+class B2sum < Formula
+  desc "BLAKE2 b2sum reference binary"
+  homepage "https://github.com/BLAKE2/BLAKE2"
+  url "https://github.com/BLAKE2/BLAKE2/archive/20160619.tar.gz"
+  sha256 "cbac833ccae56b5c6173dbeaf871aa99b32745cf7a994c7451d4533ecda55633"
+
+  def install
+    cd "b2sum" do
+      # https://github.com/BLAKE2/BLAKE2/issues/25
+      system "make", "NO_OPENMP=1"
+      bin.install "b2sum"
+      man1.install "b2sum.1"
```

Comment:
Is there no `install` target in the `Makefile`? If not, it might be worthwhile to ask upstream for one or even submit a pull request to add it.